File: server/src/main/scala/kpn/api/common/route/RouteNode.scala
package kpn.api.common.route

import kpn.api.common.LatLon

case class RouteNode(
  nodeId: Long,
  latitude: String,
  longitude: String,
  name: String,
  alternateName: String,
  longName: Option[String],
  isInWay: Boolean,
) extends LatLon

File: backend/api/src/main/java/kpn/api/common/route/RouteNode.java
package kpn.api.common.route;

import kpn.api.common.LatLon;

import java.util.Optional;

public record RouteNode(
  Long nodeId,
  String latitude,
  String longitude,
  String name,
  String alternateName,
  Optional<String> longName,
  Boolean isInWay
) implements LatLon {}
